notebook-reference/03-Markdown Cells.ipynb

```diff
@@ -0,0 +1,183 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Markdown Cells\n",
+        "\n",
+        "Markdown cells can be mixed with code cells to turn a notebook into a tutorial or just to give context for a seciton of code. For a complete set of Markdown documentation check out [markdownguide.org](https://www.markdownguide.org/) but we hope this cheat sheet will you get started.\n",
+        "\n",
+        "Click the edit markdown button ![Edit Button](https://github.com/JakeRadMSFT/csharp-notebooks/raw/main/notebook-reference/images/edit%20markdown.png) to check out the syntax.\n",
+        ""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Headers\n",
+        "# H1  \n",
+        "## H2  \n",
+        "### H3  \n",
+        "#### H4  \n",
+        "##### H5  \n",
+        "###### H6  "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Paragraphs\n",
+        "\n",
+        "This is a line in a paragraph.\n",
+        "\n",
+        "This is the next paragraph!"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Line Breaks\n",
+        "\n",
+        "Two spaces at then end of a line works.  \n",
+        "Second line."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Emphasis\n",
+        "This is a **bold text**.  \n",
+        "This is *Italicized text*.  \n",
+        "This is ~~strikethrough text~~.  \n",
+        "This is ==highlighted text==.  \n",
```

Comment:
It seems like 57-59 don't work?

Reply:
Didn't work on GitHub :) - They work in VS.